controllers/cloudstackmachine_controller.go

```diff
@@ -138,7 +138,8 @@ func (r *CloudStackMachineReconciliationRunner) ConsiderAffinity() (ctrl.Result,
 
 	agName, err := utils.GenerateAffinityGroupName(*r.ReconciliationSubject, r.CAPIMachine)
 	if err != nil {
-		r.Log.Info("getting affinity group name", err)
+		r.Log.Info("getting affinity group name", err.Error())
```

Comment:
```suggestion
```

If the error is returned and bubbled up from the the `Reconcile` method, controller-runtime will print it by default. So I don't think we need this, so I vote to just remove.

Otherwise, we can just:

```suggestion
		r.Log.Error(err, "Failed getting affinity group name")
```

Reply:
Removed the log statement. The error gets bubbled up to runtime.